packages/sources/universal-v3/src/utils/secretsDecrypter.ts

```diff
@@ -0,0 +1,46 @@
+import EthCrypto from 'eth-crypto'
+import type { Encrypted } from 'eth-crypto'
+
+export type Base64ByteString = string
+
+type HexString = string
+type Address = HexString
+type JSONstring = string
+
+export const decrypt = async (
+  privateKey: string,
+  encryptedBytesString: Base64ByteString,
+  secretsOwner: Address,
+): Promise<Record<string, unknown>> => {
+  let encryptedHexString: HexString
+  let encryptedObject: Encrypted
+  let decrypted: JSONstring
+  let decryptedPayload: { signature: string; message: string }
+  let signer: Address
```

Comment:
The only things used outside of the try catch are the signer and decrypted payload, everything else should be a const